extra_data/reader.py

```diff
@@ -1310,12 +1310,17 @@ def RunDirectory(path, include='*', file_filter=locality.lc_any):
         Function to subset the list of filenames to open.
         Meant to be used with functions in the extra_data.locality module.
     """
-    files = [f for f in os.listdir(path) if f.endswith('.h5')]
+    files = [f for f in os.listdir(path) if f.endswith('.h5') and f != 'overview.h5']
```

Comment:
I've assumed here that if we create a virtual overview file in a run directory, it will be called `overview.h5`.

Reply:
I wonder if it should not contain the proposal and run number in the name, I expect that people might want to move them around (Maybe have them all in in the same directory for convenience - maybe we want to do that as well?) and that would be convenient if they are immediately identifiable.

something like `RAW-P700000-R0001-OVERVIEW.h5` to keep something similar to the current naming scheme...

What do you think?

Reply:
I think you're probably right, but I'll see if we can get some input from ITDM.

Reply:
I've broadened this to allow for `overview` anywhere in the name. So long as we don't have a big detector called 'overview', I think this should be safe enough.